src/components/Hoverable/index.native.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * On mobile, there is no concept of hovering, so we return a plain wrapper around the component's only child.
+ */
+import React from 'react';
+import {propTypes, defaultProps} from './HoverablePropTypes';
+
+// Simply render the children, ignoring all other `Hoverable` props.
+// Using React.Children.only enforces that only a single child element is provided,
+// which matches the desired interface of `Hoverable` on web/desktop.
+const Hoverable = props => React.Children.only(props.children);
```

Comment:
Does this still work if `props.children` is a function (since it seems to matter in the web component)?